### ISSUE TITLE
[DISCUSSION] Pushing this to aurora postgres?

### ISSUE THREAD

**Issue by Dharin-shah** [cybertec-postgresql/pgfaceting#4]
Hey Maintainers,

Is it possible for us to push this extension to be supported in Aurora based postgresql.
We use aurora service for our RDS needs, and would be perfect if we'd had this extension supported there.

Thanks.

**Comment by ants** [comment]
You should first push for [pg_roaringbitmap](https://github.com/ChenHuajun/pg_roaringbitmap) availability. This extension right now does not require any superuser permissions and can deployed manually by users if need be. The performance sensitive number crunching is done using pg_roaringbitmap.

**Comment by Dharin-shah** [comment]
yes, i have mailed them to onboard both of these extensions, and have received a reply for more details of the use case, will keep posted.
